versions.json

```diff
@@ -52,5 +52,22 @@
     },
     "version": "4.1.3"
   },
-  "4.1-rc": null
+  "4.1-rc": null,
+  "4.2-rc": {
+    "alpine": {
+      "version": "3.22"
+    },
+    "openssl": {
+      "sha256": "8d1a5fc323d3fd351dc05458457fd48f78652d2a498e1d70ffea07b4d0eb3fa8",
+      "version": "3.3.4"
+    },
+    "otp": {
+      "sha256": "f696dc22812745d98a87af2aa599ffa893c7cf8dfa16be8b31db0e30d8ffa85c",
+      "version": "27.3.4.2"
+    },
+    "ubuntu": {
+      "version": "24.04"
+    },
+    "version": "4.2.0-beta.1"
+  }
```

Comment:
Should we also have a `"4.2": null` so that our `update.sh` job can give us automatic migration to releases?

Reply:
We haven't set up this repository to preserve them in that direction yet. :see_no_evil:

Reply:
Yes, but the scripts seem to have all the same bits as `mongo` so automation will give us the release, and then we just have to change [aliases](https://github.com/docker-library/rabbitmq/blob/b499e6df24978959d526acddcdc4a4842091533b/generate-stackbrew-library.sh#L4-L7), right? What is missing that wouldn't preserve the `null` in the json?